gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/PublisherApi.java

```diff
@@ -84,97 +80,80 @@
       listTopicSubscriptionsIterableCallable;
   private final ApiCallable<DeleteTopicRequest, Empty> deleteTopicCallable;
 
-  public static class ResourceNames {
-
-    // =======================
-    // ResourceNames Constants
-    // =======================
-
-    /**
-     * A PathTemplate representing the fully-qualified path to represent
-     * a project resource.
-     *
-     * <!-- manual edit -->
-     * <!-- end manual edit -->
-     */
-    private static final PathTemplate PROJECT_PATH_TEMPLATE =
-        PathTemplate.create("projects/{project}");
-
-    /**
-     * A PathTemplate representing the fully-qualified path to represent
-     * a topic resource.
-     *
-     * <!-- manual edit -->
-     * <!-- end manual edit -->
-     */
-    private static final PathTemplate TOPIC_PATH_TEMPLATE =
-        PathTemplate.create("projects/{project}/topics/{topic}");
-
-    private ResourceNames() {}
-
-    // ==============================
-    // Resource Name Helper Functions
-    // ==============================
-
-    /**
-     * Formats a string containing the fully-qualified path to represent
-     * a project resource.
-     *
-     * <!-- manual edit -->
-     * <!-- end manual edit -->
-     */
-    public static final String formatProjectPath(String project) {
-      return PROJECT_PATH_TEMPLATE.instantiate("project", project);
-    }
+  /**
+   * A PathTemplate representing the fully-qualified path to represent
+   * a project resource.
+   *
+   * <!-- manual edit -->
```

Comment:
Do you see any reason for providing this '<!-- manual edit -->` for a 'private' field?